03LessonThree/in-class-project/daily-planet.html

```diff
@@ -38,34 +42,37 @@
-                Bacon ipsum dolor amet salami bacon kielbasa biltong ham t-bone ball tip sirloin ground round fatback pork belly filet mignon alcatra leberkas. Short loin corned beef strip steak fatback kielbasa bresaola ham porchetta swine tail pig capicola filet mignon meatloaf. Pastrami flank tail turkey, kevin swine shank boudin porchetta prosciutto leberkas shankle alcatra. Turkey ball tip fatback, capicola shoulder kielbasa corned beef.
-              </p>
-              <p>
-                Beef ham hock tail, pastrami biltong cow meatloaf sirloin. Porchetta tail spare ribs fatback pig frankfurter drumstick venison leberkas sirloin. Capicola ball tip tenderloin jowl. Fatback short loin pastrami, meatloaf ground round biltong drumstick jowl kevin beef tri-tip.
-              </p>
-            </aside>
-        </main>
-        <footer>
-          <table>
+        </section>
+        <aside>
+            <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAANNAAAAJGI3NThhNjY1LTI2YzEtNDYyZi1iODJiLTY4NWQ3MGVlNDEwOA.jpg" />
+            <h3>About The Author:</h3>
+            <p>
+                Bacon ipsum dolor amet salami bacon kielbasa biltong ham t-bone ball tip sirloin ground round fatback pork belly filet mignon alcatra leberkas. Short loin corned beef strip steak fatback kielbasa bresaola ham porchetta swine tail pig capicola filet mignon
+                meatloaf. Pastrami flank tail turkey, kevin swine shank boudin porchetta prosciutto leberkas shankle alcatra. Turkey ball tip fatback, capicola shoulder kielbasa corned beef.
+            </p>
+            <p>
+                Beef ham hock tail, pastrami biltong cow meatloaf sirloin. Porchetta tail spare ribs fatback pig frankfurter drumstick venison leberkas sirloin. Capicola ball tip tenderloin jowl. Fatback short loin pastrami, meatloaf ground round biltong drumstick jowl
+                kevin beef tri-tip.
+            </p>
+        </aside>
+    </main>
+    <footer>
+        <table>
             <tr>
-              <td>
-                &copy The Daily Planet 2016
+                <td>
+                    &copy The Daily Planet 2016
+                </td>
+                <td>
+                    <a href="https://www.facebook.com/ESPN/?fref=ts" />
+                    <img src="http://www.freelargeimages.com/wp-content/uploads/2014/11/Facebook_logo-7.png" />
+                    </a>
+                </td>
+                <td>
+                    kjxfknjnjnfjcnvjnvmkdjfdfjdfbjdnj
                 </td>
-              <td>
-                <a href="https://www.facebook.com/ESPN/?fref=ts"/>
-                  <img src="http://www.freelargeimages.com/wp-content/uploads/2014/11/Facebook_logo-7.png"/>
-                </a>
-              </td>
-              <td>
-                kjxfknjnjnfjcnvjnvmkdjfdfjdfbjdnj
-              </td>
             </tr>
-          </table>
-        </footer>
-    </body>
+        </table>
+    </footer>
+</body>
+
 </html>

```

Comment:
Great work, Steve! You completed all the requirements for the assignment and still took it a step or two beyond. You linked to real social media profiles, and used a table to create columns. Well done.